doc/faq.rst

```diff
@@ -65,28 +65,29 @@ optimized file format such as HDF5 to reduce data load times. Various libraries
 such as H5Py, PyTables and pandas provides a Python interface for reading and
 writing data in that format.
 
-Can I add this new algorithm that I (or someone else) just published?
--------------------------------------------------------------------------
-No. As a rule we only add well-established algorithms. A rule of thumb is at least
-3 years since publications, 200+ citations and wide use and usefullness. A
+What are the inclusion criteria for new algorithms ?
```

Comment:
@amueller should it be `algorithms`? We could make this generically applicable for any new feature right?